system/HTTP/Files/UploadedFile.php

```diff
@@ -75,13 +82,15 @@ class UploadedFile extends File implements UploadedFileInterface
      *
      * @param string $path         The temporary location of the uploaded file.
      * @param string $originalName The client-provided filename.
+     * @param string $clientPath   The webkit relative path of the uploaded file.
      * @param string $mimeType     The type of file as provided by PHP
      * @param int    $size         The size of the file, in bytes
      * @param int    $error        The error constant of the upload (one of PHP's UPLOADERRXXX constants)
      */
-    public function __construct(string $path, string $originalName, ?string $mimeType = null, ?int $size = null, ?int $error = null)
+    public function __construct(string $path, string $originalName, ?string $clientPath, ?string $mimeType = null, ?int $size = null, ?int $error = null)
```

Comment:
add param in the middle is BC break.

Reply:
Ah, yeah. Perhaps I should move the param to the end for now, so it doesn't interfere with any instances where I didn't update the arguments.

Reply:
Yes, this must be added in the last.

Reply:
And I forgot to allow that param as null